src/internal/utils/date-time/__tests__/format-date-time-with-offset.test.ts

```diff
@@ -0,0 +1,168 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This file is an adaptation of `format-date-range.test.ts` because `formatDateRange` doesn't exist anymore; instead, we call `formatDateTimeWithOffset` twice, once for the start date and once for the end date, which are rendered together at the component level.